azerothcore/azerothcore-wotlk
# fix(Script/Instance): Ahn'kahet: The Old Kingdom - Jedoga Shadowseeke sacrifice

## Changes Proposed:
-  Moved sacrifice to an event and deleted `spell_jedoga_sacrafice_beam` spell script.

## Issues Addressed:

- Closes #10854
- Closes #12836
- Closes #13071
- Closes #14748

## SOURCE:


## Tests Performed:

- tested

## How to Test the Changes:


set to HC 5man
.go c id 29310
bring boss to under 55%
Let an add sacrifice itself, check for achievement

## Known Issues and TODO List:


- [ ] [(COSMETIC)](https://youtu.be/C0YIMouZCXs?t=41) Sacrifice Beam visual was, and still is broken. 


## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.
